### ISSUE TITLE
[Announcement] Project 2 Design Review Time Arrangement

### ISSUE THREAD

**Issue by t-hyun** [swsnu/osspr2022#49]
Hello there :) This is an announcement about project 2 design review - from 3/28(Mon) to 4/2(Sat).
Select one time slot on doodle link.
If you with an offline meeting, mail to osspr.2022.ta@gmail.com.

Team 01~08: TA Minjae Kim
Appointment setting: [https://doodle.com/meeting/participate/id/zbqNyYkd](https://doodle.com/meeting/participate/id/zbqNyYkd)
Zoom Link: [https://snu-ac-kr.zoom.us/j/9602971615](https://snu-ac-kr.zoom.us/j/9602971615) (Meeting ID:960 297 1615)

Team 09~17: TA Taehyun Lee
Appointment setting: [https://doodle.com/meeting/participate/id/QeZjPx2e](https://doodle.com/meeting/participate/id/QeZjPx2e)
Zoom Link: [https://snu-ac-kr.zoom.us/j/3456856440](https://snu-ac-kr.zoom.us/j/3456856440) (Meeting ID: 345 685 6440)

**Comment by sunhongmin225** [comment]
Is it okay to check ✅ for all the multiple options that I'm available?

**Comment by widiba03304** [comment]
@sunhongmin225 Yes, but make sure that you and your team members are all available at those times!

**Comment by t-hyun** [comment]
Added more time slots on Thu & Fri. Check if those times are better!
